File: app/src/main/java/com/epigram/android/data/FastScroller.java
package com.epigram.android.data;

import android.content.Context;
import android.util.DisplayMetrics;
import androidx.recyclerview.widget.LinearSmoothScroller;

public class FastScroller extends LinearSmoothScroller {

    @Override
    protected float calculateSpeedPerPixel(DisplayMetrics displayMetrics) {
        return super.calculateSpeedPerPixel(displayMetrics)*0.1f;
    }

    public FastScroller(Context context) {
        super(context);
    }
}

File: app/src/main/java/com/example/epigram/arch/android/FastScroller.kt
package com.example.epigram.arch.android

import android.content.Context
import android.util.DisplayMetrics
import androidx.recyclerview.widget.LinearSmoothScroller

class FastScroller(context: Context) : LinearSmoothScroller(context) {

    override fun calculateSpeedPerPixel(displayMetrics: DisplayMetrics): Float {
        return super.calculateSpeedPerPixel(displayMetrics) * 0.1f
    }
}


